truffle-config.js

```diff
@@ -61,6 +60,12 @@ module.exports = {
       network_id: "*",       // Any network (default: none)
       disableConfirmationListener: true,
     },
+    test2: {
+      host: "127.0.0.1",     // Localhost (default: none)
```

Comment:
are not they suppose to be different?

Reply:
Not really. Both are basically targeting localhost. Just needed to satisfy the form to be able to use the --network param.